jobbergate-cli/jobbergate_cli/exceptions.py

```diff
@@ -105,3 +107,28 @@ def wrapper(*args, **kwargs):
             raise typer.Exit(code=1)
 
     return wrapper
+
+
+def handle_authentication_error(func):
```

Comment:
Since `handle_abort` is already being used on all the commands where this handler would need to be included, why not bake this logic directly into `handle_abort` and avoid having to use an extra decorator?

Reply:
I initially thought that making both explicit was better than making them implicit, but this duplication on all commands is not ideal. I modified the code a bit to centralize the error handling: f706334.